### ISSUE TITLE
2.5.0-development - page.render does not render transparent background

### ISSUE THREAD

**Issue by smagnetize** [ariya/phantomjs#14759]
1. Which version of PhantomJS are you using? Tip: run `phantomjs --version`.
2.5.0-development 2017-01-08

2. What steps will reproduce the problem?
  Code:
```
var webPage = require('webpage');
var page = webPage.create();
page.open('https://www.google.co.uk/images/branding/googlelogo/2x/googlelogo_color_120x44dp.png', function (status) {
  page.render('test.png', { format: 'png' });
  phantom.exit();
});
```
  **Expected:** test.png to have a transparent background (like 2.1.1 does)
  **Actual:** test.png has solid white background

3. Which operating system are you using?
Windows 2008, Windows 10

4. Did you use binary PhantomJS or did you compile it from source?
binaries

**Comment by jierwang** [comment]
I encountered the same issue. The transparent background of the canvas (pixel rgba value as 0, 0, 0, 0) becomes solid (0, 0, 0, 255) after rendered to png via page.render(). I double checked. I got the imageData of the canvas via context.getImageData(0, 0, canvas.width, canvas.height) before page.render and the rgba value of the first pixel contained in imageData.data are (0,0,0,0). However in the generated png file, same position, alpha changes from 0 to 255. Any suggestion? Thanks a lot.

phantomjs version: 2.1.1
OS: windows 7
